### ISSUE TITLE
Keyboard shortcut: Show/hide accounts with zero balance

### ISSUE THREAD

**Issue by Zburatorul** [beancount/fava#1128]
Fava has the `show-acounts-with-zero-balance` option, but it can only be changed at start-up (or at least requires writing to a file, and then refreshing). I would like a keyboard shortcut that can show/hide the accounts on the fly.

**Comment by redstreet** [comment]
I've run into the need for this too at various times. @Zburatorul, I'd imagine a PR would be appreciated if you are up for it.

**Comment by Zburatorul** [comment]
How do I get started on this?

**Comment by redstreet** [comment]
@yagebu or others should be able to help guide.

**Comment by yagebu** [comment]
I don't know whether this would be feasible to implement with the way that this option is currently provided, as we'd have to rewrite the option in the input file to change. One way to implement this might be to move the option from the Beancount file to be stored in the browser. Then an interface to change this option could then be more easily added in the frontend (which could then also get a keyboard shortcut).

To me it seems that this is an option that, as a display option, could work well as an automatically stored option stored in the browser. What do you think?

**Comment by redstreet** [comment]
The use case I've run into for this is the occasional, temporary toggle. For example: I have the occasional need to view a closed account, or look up the journal of a credit card with a zero balance.

I'd imagine that is the primary use case for most users. Keeping that in mind, a suggestion is to retain the input file config option, which is read and respected at fava startup, and beyond that, allow per-session toggles using a keyboard shortcut. The toggled state is never persisted beyond the session. WDYT?

**Comment by Zburatorul** [comment]
I agree with this assessment @redstreet, and my use case happens to coincide with yours.

**Comment by redstreet** [comment]
@yagebu, WDYT? If you agree, would you be able to provide @Zburatorul pointers to implement?
